solutions/fscloud/variables.tf

```diff
@@ -0,0 +1,74 @@
+variable "ibmcloud_api_key" {
+  description = "The IBM Cloud platform API key needed to deploy IAM enabled resources."
+  type        = string
+  sensitive   = true
+}
+
+variable "region" {
+  type        = string
+  description = "The IBM Cloud region where the Cloudant instance will be provisioned."
+  default     = "us-south"
+}
+
+variable "resource_group_id" {
+  description = "The Id of an existing IBM Cloud resource group where the instance will be grouped."
+  type        = string
+  default     = null
+}
+
+variable "existing_resource_group" {
+  type        = bool
+  description = "If existing resource group will be used."
+  default     = false
+}
+
+variable "resource_group_name" {
+  type        = string
+  description = "New resource group name to use."
+  default     = null
+}
+
+variable "instance_name" {
+  description = "The name of the Cloudant instance"
+  type        = string
+}
+
+variable "include_data_events" {
+  type        = bool
+  description = "Include data event types in events sent to IBM Cloud Activity Tracker. If set to false, only management events will be sent to Activity Tracker."
+  default     = false
```

Comment:
Do you know why we default to false here? Wondering if the better practise is to default to `true`. What do the docs recommend? What will Dreadnought be doing?